site/content/docs/main/upgrade-to-1.10.md

```diff
@@ -17,12 +17,15 @@ If you're not yet running at least Velero v1.6, see the following:
 - [Upgrading to v1.6][6]
 - [Upgrading to v1.7][7]
 - [Upgrading to v1.8][8]
+- [Upgrading to v1.9][9]
```

Comment:
1. Prerequisite is wrong.
2. I'm not sure if it's necessary to list all the previous minor versions upgrade instructions.

Reply:
What about we just keep 3 or 5 items on the list?

Reply:
@OrlinVasilev Do you think how many items should we keep for upgrading instructions?